src/brogue/RogueMain.c

```diff
@@ -950,12 +951,11 @@ void gameOver(char *killedBy, boolean useCustomPhrasing) {
     if (player.bookkeepingFlags & MB_IS_DYING) {
         // we've already been through this once; let's avoid overkill.
         return;
-    } else {
-        player.bookkeepingFlags |= MB_IS_DYING;
```

Comment:
why is this done?

Reply:
Because
```
if (...) {
  return;
} else {
  a;
}
b;
c;
```
is the same as
```
if (...) {
  return;
}
a;
b;
c;
```
but the latter is cleaner?

Reply:
oh lol, I view diffs side by side and didn't see the corresponding addition, my bad :grin: